tests/baselines/reference/typesVersions.ambientModules.trace.json

```diff
@@ -32,6 +32,10 @@
     "File 'tests/cases/conformance/moduleResolution/node_modules/ext/ts3.1/other.ts' does not exist.",
     "File 'tests/cases/conformance/moduleResolution/node_modules/ext/ts3.1/other.tsx' does not exist.",
     "File 'tests/cases/conformance/moduleResolution/node_modules/ext/ts3.1/other.d.ts' does not exist.",
+    "'package.json' has a 'typesVersions' entry '>=3.1.0-0' that matches compiler version '3.1.0-dev', looking for a pattern to match module name 'index'.",
```

Comment:
Looks like this message will change everytime we change the compiler version (because of version of compiler being in the message) so if we forget about it every version change will introduce test failure?

Reply:
You're right. Not sure what we can do about this besides change the message.

Reply:
You can use `sanitizeTraceResolutionLogEntry` in harness/utils.ts to normalize the message for test runs. Its already in use in the compiler/conformance tests.

Reply:
Rather, its already being used here.

Reply:
So nothing to do? @sheetalkamat @rbuckton Good to merge?